fighthealthinsurance/common_view_logic.py

```
@@ -155,12 +155,20 @@ async def _convert_input(self, input_path: str) -> Optional[str]:
                         new_input_path = f"{input_path}.u8.txt"
                     except:
                         pass
+                if input_path.endswith(".html"):
+                    html_command = base_convert_command
+                    html_command.extend("-thtml")
+                    try:
+                        await check_call(html_command)
+                        return f"{input_path}.pdf"
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)